docs/config/index.md

```diff
@@ -507,6 +507,12 @@ export default async ({ command, mode }) => {
 
   Options to pass on to [@rollup/plugin-commonjs](https://github.com/rollup/plugins/tree/master/packages/commonjs).
 
+  ### build.commonjsOptions
```

Comment:
Wrong name and wrong indentation here.

Reply:
Sometimes, I feel so stupid when I look at the things I did some weeks ago ... 🙈 
Is fixed now.